antlr_ast.py

```diff
@@ -170,11 +178,10 @@ def visitor(self, ctx):
     return visitor
 
 
-def bind_to_visitor(visitor_cls, node_cls, rule_name, method):
+def bind_to_visitor(visitor_cls, rule_name, visitor):
     """Assign AST node class constructors to parse tree visitors."""
-    visitor = get_visitor(node_cls, method)
     # TODO raise warning if attr already on visitor?
-    method_name = "visit{}".format(rule_name.capitalize())
+    method_name = "visit{}".format(rule_name[0].upper() + rule_name[1:])
```

Comment:
Extracting this as a `util` function and adding a test would make the desired behaviour explicit.